extensions/libraries/redcore/database/sqlparser/sqlparser.php

```diff
@@ -1071,7 +1071,7 @@ private function process_from(&$tokens)
 					}
 					$parseInfo['alias']['name']      = $str;
 					$parseInfo['alias']['base_expr'] = trim($parseInfo['alias']['base_expr']);
-					continue;
+					continue 2;
```

Comment:
take notice that with continue 2 you jump out of the loop, that may seem to fix some issue, but that is just because you skip everything altogether. Continue 2 should not be used unless you want to jump out on purpose. And in your cases from this PR, I dont see a valid reason to jump out.

Reply:
@Kixo @VKattou @nhungredweb 
Please read https://stackoverflow.com/questions/26431066/continue-2-and-break-in-switch-statement
the switch clause in this situation is also treated as a loop.
So here for eg, theres a switch inside a foreach inside a function, 
the inner 'loop' is the switch, the outer loop is the foreach, and "continue 2" continues out to the foreach

Reply:
yes, but did you checked the code that is then ignored after the switch command if you do a continue 2;, some code is now skipped if you do a continue 2 instead of continue, and especially lines that have explicitly stated so like: `continue; // ends the switch statement!` 
https://github.com/redCOMPONENT-COM/redCORE/pull/857/files#diff-63df9c21944e72b04ec542238e027e05L1150